### PR TITLE
README.md

### PR DESCRIPTION
**Pull request by keshavgoyal97**
Put some changes in the "Readme.md" file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@ git + hub = github
 ==================
 
 hub is a command line tool that wraps `git` in order to extend it with extra
-features and commands that make working with GitHub easier.
+features and commands, that makes it easier to work with github.
```
Comment (mislav) [inline README.md]: I don't think this wording change is necessary.

File: README.md [hunk 2]
```diff
@@ -24,6 +24,9 @@ Dependencies:
 
 #### Homebrew
 
+can install homebrew from the below link
+"https://docs.brew.sh/Installation"
```
Comment (mislav) [inline README.md]: It's a good idea to link to Homebrew. However, I don't think we need to add an additional sentence like this. How about turning “Homebrew” in the existing sentence into a link?

```
`hub` can be installed through [Homebrew](https://brew.sh) on macOS:
```

**Comment by keshavgoyal97** [comment]
I changed readme file accordingly.

File: README.md [hunk 3]
```diff
@@ -2,7 +2,7 @@ git + hub = github
 ==================
 
 hub is a command line tool that wraps `git` in order to extend it with extra
-features and commands that make working with GitHub easier.
+features and commands, that make working with GitHub easier.
```
Comment (giuscri) [inline README.md]: -1

**Comment by keshavgoyal97** [comment]
What?

**Comment by giuscri** [comment]
adding comma is not improving english imho

**Comment by keshavgoyal97** [comment]
It was not intentional.

**Comment by mislav** [comment]
Thank you!

**Pull request merged**